### PR TITLE
Add docker and crictl to offline-installation-tool.sh

### PR DESCRIPTION
**Pull request by kinglong08**
Solved this issue：https://github.com/kubesphere/ks-installer/blob/528c292b30efb2b3bf4a05d6349a150c559e99bc/scripts/offline-installation-tool.sh#L110

### CODE REVIEW

**Comment by kinglong08** [comment]
> /assign @24sama

File: scripts/offline-installation-tool.sh [hunk 1]
```diff
@@ -108,6 +110,10 @@ if [[ ${binary} == "true" ]]; then
      curl -L -o ${binariesDIR}/cni-plugins-linux-${ARCH}-${CNI_VERSION}.tgz https://containernetworking.pek3b.qingstor.com/plugins/releases/download/${CNI_VERSION}/cni-plugins-linux-${ARCH}-${CNI_VERSION}.tgz
      echo "Download etcd ..."
      curl -L -o ${binariesDIR}/etcd-${ETCD_VERSION}-linux-${ARCH}.tar.gz https://kubernetes-release.pek3b.qingstor.com/etcd/release/download/${ETCD_VERSION}/etcd-${ETCD_VERSION}-linux-${ARCH}.tar.gz
+     echo "Download crictl ..."
+     curl -L -o ${binariesDIR}/crictl-${CRICTL_VERSION}-linux-${ARCH}.tar.gz https://kubernetes-release.pek3b.qingstor.com/cri-tools/releases/download/${CRICTL_VERSION}/crictl-${CRICTL_VERSION}-linux-${ARCH}.tar.gz
+     echo "Download docker ..."
+     curl -L -o ${binariesDIR}/docker-${DOCKER_VERSION}.tgz https://mirrors.aliyun.com/docker-ce/linux/static/stable/x86_64/docker-${DOCKER_VERSION}.tgz
```
Comment (24sama) [inline scripts/offline-installation-tool.sh:116]: The `x86_64` should be the `${ARCH}`.

File: scripts/offline-installation-tool.sh [hunk 2]
```diff
@@ -121,6 +127,10 @@ if [[ ${binary} == "true" ]]; then
      curl -L -o ${binariesDIR}/cni-plugins-linux-${ARCH}-${CNI_VERSION}.tgz https://github.com/containernetworking/plugins/releases/download/${CNI_VERSION}/cni-plugins-linux-${ARCH}-${CNI_VERSION}.tgz
      echo "Download etcd ..."
      curl -L -o ${binariesDIR}/etcd-${ETCD_VERSION}-linux-${ARCH}.tar.gz https://github.com/coreos/etcd/releases/download/${ETCD_VERSION}/etcd-${ETCD_VERSION}-linux-${ARCH}.tar.gz
+     echo "Download crictl ..."
+     curl -L -o ${binariesDIR}/crictl-${CRICTL_VERSION}-linux-${ARCH}.tar.gz https://github.com/kubernetes-sigs/cri-tools/releases/download/${CRICTL_VERSION}/crictl-${CRICTL_VERSION}-linux-${ARCH}.tar.gz
+     echo "Download docker ..."
+     curl -L -o ${binariesDIR}/docker-${DOCKER_VERSION}.tgz https://download.docker.com/linux/static/stable/x86_64/docker-${DOCKER_VERSION}.tgz
```
Comment (24sama) [inline scripts/offline-installation-tool.sh:133]: Here is also the x86_64 should be the ${ARCH}.

**Review by 24sama — CHANGES_REQUESTED** [review]

**Comment by 24sama** [comment]
@kinglong08 
Thanks for your contribution!

**Review by kinglong08 — COMMENTED** [review]
x86_64   <=>  ${ARCH}

**Review by 24sama — APPROVED** [review]

**Pull request merged**